trusttrees/__main__.py

```diff
@@ -21,12 +21,8 @@ def main(command_line_args=sys.argv[1:]):
     if args.target_hostname:
         target_hostnames = [args.target_hostname]
     else:
-        targets = open(args.target_hostnames_list)
```

Comment:
Very pythonic, thanks @alanyee!

Want to make the same change in
https://github.com/mandatoryprogrammer/TrustTrees/blob/ca8173d052c4973ce1fc6277e18248e40a8d0ff7/setup.py#L9-L15

Reply:
Done 👍